examples/kubernetes/README.rst

```diff
@@ -0,0 +1,58 @@
+Kubernetes Deployment
+=====================
+
+This directory contains all Cilium deployment files that can be used in
+Kubernetes.
+
+Each directory represents a Kubernetes version, from :code:`1.7` to :code:`1.11`,
+and inside each version there is a list of files to deploy Cilium.
+
+The structure directory will be :code:`${k8s_major_version}.${k8s_minor_version}/*.yaml`.
+
+To generate those files simply run :code:`make`, which will pick up the Cilium
+version from the :code:`VERSION` file at the root of the repo, or if you want to
+specify the Cilium version yourself use :code:`make CILIUM_VERSION=X.Y.Z`.
+
+If you want to clean up a specific version, run :code:`make CILIUM_VERSION=X.Y.Z`
+which will delete all generated files for a specific Cilium version
```

Comment:
How does this work? The makefile just clears things up and CILIUM_VERSION is used only to populate the image version, right?

Reply:
cool. It would delete the copy checked out, right?